server/controllers/RecipeController.js

```diff
@@ -0,0 +1,96 @@
+import { Recipes as RecipeClass } from '../helpers/in-memory';
+
+/**
+ * Handles Recipe request operations
+ * @class RecipeController
+ * @param { null } void
+ */
+class RecipeController extends RecipeClass {
+  /**
+   * Adds a new recipe
+   * @memberof RecipeController
+   * @param { object } req -request
+   * @param { object } res -respone
+   * @returns { obejct } server response
+   */
+  addRecipe(req, res) {
+    return super.create({...req.body })
+      .then(recipe => res.status(201).send({
+        recipe,
+        message: `${recipe.name} added successfully`
+      }))
+      .catch(error => res.status(500).send({
```

Comment:
You'd also want to display errors aside from 500 errors to your users.

For example, 400 errors like validation error messages

Reply:
fixed https://github.com/hephzaron/youth-interact/pull/1/commits/40a800387648a628b9f90d30ae10b6a22c3fea42